internal/services/producer.go

```diff
@@ -65,3 +68,13 @@ func (p *Producer) Stop() {
 	}
 	log.Printf("Producer closed")
 }
+
+func canaryMessageValueToJson(producerID string, index int, timestamp int64) string {
+	m := CanaryMessage{
+		ProducerID: producerID,
+		MessageID:  index,
+		Timestamp:  timestamp,
+	}
+	json, _ := json.Marshal(m)
```

Comment:
is it safe for us to ignore the error here?

Reply:
we are building the message, so we would expect valid values there but anyway we can think more about that adding some more error handling later.